### PR TITLE
Add example for registering multiple custom mark

### LINKED ISSUE #7880: docs: Explain how to register multiple custom markers
In the [docs on registering markers](https://docs.pytest.org/en/latest/example/markers.html#registering-markers), it explains how to register a single custom marker but not how to register multiple custom markers.

It would be great if there were a sentence like, "To register multiple custom markers, put each one on its own line in your `pytest.ini` file," with or without an example.

### PR DESCRIPTION
**Pull request by gutsytechster**
This fixes #7880

### CODE REVIEW

**Comment by gutsytechster** [comment]
Let me know if this requires an entry in the changelog. :)

**Review by bluetech — COMMENTED** [review]
Thanks @gutsytechster!

To keep things a bit shorter, how about adding a second marker to the first example, instead of adding a second example? That, I think, would be sufficient for getting across how multiple markers are registered.

File: doc/en/example/markers.rst [hunk 1]
```diff
@@ -222,6 +222,16 @@ Registering markers for your test suite is simple:
...
+
+.. code-block:: ini
+
+    # content of pytest.ini
+    [pytest]
+    markers =
+        webtest: mark a test as a webtest.
+        slow: mark test as slow.
+
 You can ask which markers exist for your test suite - the list includes our just defined ``webtest`` markers:
```
Comment (bluetech) [inline doc/en/example/markers.rst:235]: This sentence (and the next code block) would need a little update as well.

**Review by bluetech — APPROVED** [review]
Thanks!

**Pull request merged**